test/integ/quote-gouda.test.ts

```diff
@@ -447,6 +448,82 @@ describe('quoteUniswapX', function () {
           }
         });
 
+        it(`ETH -> large cap, large trade should return valid quote`, async () => {
+          const amount = await getAmount(1, type, 'ETH', 'UNI', '1');
+          const quoteReq: QuoteRequestBodyJSON = {
+            requestId: 'id',
+            tokenIn: NATIVE_ADDRESS,
+            tokenInChainId: 1,
+            tokenOut: UNI_MAINNET.address,
+            tokenOutChainId: 1,
+            amount: amount,
+            type,
+            slippageTolerance: SLIPPAGE,
+            configs: [
+              {
+                routingType: RoutingType.DUTCH_LIMIT,
+                swapper: alice.address,
+                useSyntheticQuotes: true,
+              },
+            ] as RoutingConfigJSON[],
+          };
+
+          const response = await axios.post<QuoteResponseJSON>(`${API}`, quoteReq);
+          const {
+            data: { quote },
+            status,
+          } = response;
+
+          const routingResponse = await axios.get<RoutingApiQuoteResponse>(
+            `${ROUTING_API}?${qs.stringify({
+              tokenInAddress: 'ETH', // Routing API doesn't support 0x0 as native
+              tokenOutAddress: UNI_MAINNET.address,
+              tokenInChainId: 1,
+              tokenOutChainId: 1,
+              amount: amount,
+              type: type === 'EXACT_INPUT' ? 'exactIn' : 'exactOut',
+              recipient: alice.address,
+              slippageTolerance: SLIPPAGE,
+              deadline: '360',
+              algorithm: 'alpha',
+              enableUniversalRouter: true,
+            })}`
+          );
+          expect(routingResponse.status).to.equal(200);
+
+          const order = new DutchOrder((quote as any).orderInfo, 1);
+          expect(status).to.equal(200);
+          // account for gas and slippage
+          expect(order.info.swapper).to.equal(alice.address);
+          expect(order.info.outputs.length).to.equal(1);
+
+          const { tokenInBefore, tokenInAfter, tokenOutBefore, tokenOutAfter } = await executeSwap(
+            order,
+            Ether.onChain(1),
+            UNI_MAINNET
+          );
+
+          if (type === 'EXACT_INPUT') {
+            // We check the *wrapped* balance, since Gouda acts on wrapped tokens, and since we don't
+            // wrap ETH in this test. We just use Alice's pre-existing WETH balance.
+            expect(tokenInBefore.subtract(tokenInAfter).toExact()).to.equal('1');
+            checkQuoteToken(
+              tokenOutBefore,
+              tokenOutAfter,
+              CurrencyAmount.fromRawAmount(UNI_MAINNET, order.info.outputs[0].startAmount.toString())
+            );
+          } else {
+            expect(
+              tokenOutAfter.subtract(tokenOutBefore).greaterThan(1) || tokenOutAfter.subtract(tokenOutBefore).equalTo(1)
```

Comment:
Strangely on exact output, the token out balance is changed by more than 1 (its 1.01).  IIRC in classic this can happen due to rounding, but can't see why this would happen for a Gouda swap